### DISCUSSION TITLE
"What's the surface here" question for non-existent (but required) way

### DISCUSSION THREAD

**Discussion by jamescridland** [streetcomplete/StreetComplete#5350] [Feedback]
While wandering around Munich recently, [I was asked about the surface for a non-existent way](https://www.openstreetmap.org/note/3962914). The way exists, I was told, for routing purposes; but it isn't visible on the ground.

> If such is asked in Streetcomplete, best ignore it and walk on. The app cannot judge, whether the question is meaningful for the particular way, so the mapper must decide.

I would typically create a note when something doesn't match with the actual on-the-ground experience. That then allows me to go back and make edits where it makes sense to do so.

In this case, I'm wondering whether there's a way to programmatically spot this type of way, to avoid polluting the map. But there again, I guess these types of ways are polluting the map anyway - given they only really exist for the router. It might be worthwhile setting a minimum length required for a way?

**Comment by Helium314** [comment]
Looks like the way could be tagged as `link`, then SC will not ask: https://wiki.openstreetmap.org/wiki/Key:footway
